### DISCUSSION TITLE
Resolving Triple Login Requirement with Safe Exam Browser  and Moodle

### DISCUSSION THREAD

**Discussion by Elrashid** [SafeExamBrowser/seb-win-refactoring#702] [Q&A]
Hello Community,

I'm reaching out for help with an issue we're experiencing with Safe Exam Browser (SEB) integration within our Moodle setup. Our students are being asked to log in three times during the process of accessing an exam:

1. Initial Moodle Login: Students first log in to Moodle via a standard web browser.
2. Launching SEB-Win: They then click a link to launch SEB, which prompts for login credentials again.
3. SEB-Win Restart: After the initial SEB launch, SEB restarts for some reason and asks for the login details a third time.

Any advice, suggestions, or guidance on this matter would be greatly appreciated. If additional information is needed, please let me know, and I will provide it promptly.

**Comment by dbuechel** [comment]
The first and second login are unfortunately outside of our control (albeit that Moodle resp. any other LMS could implement an auto-login mechanism on their side to avoid the second login). But you can get rid of the third login by disabling the clearing of the browser session when starting under "Session Handling" in tab "Exam" of the configuration tool: https://safeexambrowser.org/windows/win_usermanual_en.html#ExamPane.

We however recognized that having to manually configure the option for this scenario is not clear to most users, so we are currently looking into a possibility to improve the user experience on our side by automatically enabling resp. overriding this option for that particular use case.

**Comment by dbuechel** [comment]
We shall track this issue under #703.
